api/src/main/java/org/openmrs/module/metadatamapping/MetadataTermMapping.java

```diff
@@ -44,7 +44,7 @@ private MetadataTermMapping() {
 	 * Construct a new metadata term mapping.
 	 * @param metadataSource defines the namespace of this term, may not be null
 	 * @param metadataTermCode code of this term within metadataSource, may not be null
-	 * @param referredObject object this term maps to, may not be null, must have {@link OpenmrsMetadata#getUuid()}
+	 * @param referredObject object this term maps to, may be null, if exists must have {@link OpenmrsMetadata#getUuid()}
 	 * @since 1.1
 	 */
 	public MetadataTermMapping(MetadataSource metadataSource, String metadataTermCode, OpenmrsMetadata referredObject) {
```

Comment:
Any chance you could add a constructor which takes just metadatsource and metadataTermCode?